app/server/src/controllers/sessionsController.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from "express";
+import { AppLocals } from "../types";
+import { SessionStore } from "../db/sessionStore";
+
+export class SessionsController {
+    static postSession = async (req: Request, res: Response) => {
+        const { redis, wodinConfig } = req.app.locals as AppLocals;
+        const { appName, id } = req.params;
+        const store = new SessionStore(redis, wodinConfig.courseTitle, appName);
```

Comment:
rather than courseTitle here, probably best to add another property which is less variable and contains no spaces? Could just be called `savePrefix` to be really explicit